how-to/customize-workspace/docs/how-to-define-apps.md

```diff
@@ -163,6 +163,61 @@ The configuration above shows that it doesn't enable all of the manifest types t
 
 These endpoints are using the built in fetch support and the options are passed to fetch options in order to get back the JSON list of app entries.
 
+### How would I map a feed of custom or standard based apps?
+
+**The first option to consider is whether you want to do a conversion server side.**
+
+If you need to map an FDC3 1.2 / 2.0 App Entry so that it can be used by the platform then we have an endpoint module and a pattern to support it (module and endpoint ids can be anything you want).
+
+You would define two endpoints. One endpoint takes the source apps and the second app transforms that into the platform App shape you have seen above.
+
+```json
+"endpointProvider": {
+    "modules": [
+        {
+            "enabled": true,
+            "id": "fdc3-app",
+            "url": "http://localhost:8080/js/modules/endpoints/fdc3-app.bundle.js"
+        }
+    ],
+    "endpoints": [
+        {
+     "id": "fdc3-in",
```

Comment:
Formatting looks off here

Reply:
yeah that was the vscode formatting after linting complained.